.github/workflows/build_tag.yml

```diff
@@ -34,10 +34,8 @@ jobs:
       run: |
         ./hack/build-antrea-linux-all.sh --pull
         echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-        docker tag antrea/antrea-ubuntu:"${VERSION}" antrea/antrea-ubuntu-amd64:"${VERSION}"
```

Comment:
Should we also removed the line here? 
https://github.com/antrea-io/antrea/blob/main/ci/jenkins/mellanox/scripts/start_ci.sh#L114

Reply:
I will remove the line, but I think we should consider removing the script... as it is no longer maintained AFAIK

Reply:
The script is definitely broken too, it still uses the Harbor name for the Antrea docker images